README.md

```diff
@@ -154,6 +154,11 @@ UTS namespace ([CLONE_NEWUTS](http://linux.die.net/man/2/clone)): The sandbox wi
 
 Seccomp filters: You can pass in seccomp filters that limit which syscalls can be done in the sandbox. For more information, see [Seccomp](https://en.wikipedia.org/wiki/Seccomp).
 
+If you are not filtering out `TIOCSTI` commands using seccomp filters,
+argument `--new-session` is needed to protect against out-of-sandbox
+command execution
+(see [CVE-2017-5226](https://github.com/containers/bubblewrap/issues/142)).
```

Comment:
I think this is the most important part of what's here.

Reply:
Strictly speaking this is only true if bubblewrap is run with a controlling terminal, but it's remarkably difficult to not have a controlling terminal.

Reply:
@smcv I'm reading that as an informational note but with a tendency towards keeping as is (for simplicity and "good enough already").